src/runtime.js

```diff
@@ -525,13 +523,12 @@ function prepareAndRegisterServiceForDoc(global, extensions,
  * @param {!Window} global
  * @param {!./service/extensions-impl.Extensions} extensions
  * @param {string} name
- * @param {function(new:Object, !./service/ampdoc-impl.AmpDoc)=} opt_ctor
- * @param {function(!./service/ampdoc-impl.AmpDoc):!Object=} opt_factory
+ * @param {!function(new:Object, !./service/ampdoc-impl.AmpDoc)} ctor
```

Comment:
Oops. Do we need to think through backward compatibility? We usually provide the backward compatibility between two PROD versions. E.g. new v0.js should work with the old amp-carousel.js